pages/_app.jsx

```diff
@@ -20,6 +20,7 @@ Sentry.init({
 // eslint-disable-next-line react/prop-types
 function MyApp({ Component, pageProps }) {
   useEffect(() => {
+    // eslint-disable-next-line no-undef
```

Comment:
for what reason this comment added?

Reply:
There was problem with eslint, needed to add that so that it would work.